internal/controller/controllers/agent_controller_test.go

```diff
@@ -2727,6 +2738,41 @@ VU1eS0RiS/Lz6HwRs2mATNY5FrpZOgdM3cI=
 			getNodeCount:        1,
 			bmhExists:           true,
 		},
+		{
+			name:         "Do not auto approve CSR for baremetal host with MAPI",
+			createClient: true,
+			hostname:     CommonHostname,
+			node: &corev1.Node{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: CommonHostname,
+				},
+				Status: corev1.NodeStatus{
+					Conditions: []corev1.NodeCondition{
+						{
+							Type:   corev1.NodeReady,
+							Status: corev1.ConditionFalse,
+						},
+					},
+					Addresses: []corev1.NodeAddress{
+						{
+							Type:    corev1.NodeInternalIP,
+							Address: "192.168.111.28",
+						},
+					},
+				},
+			},
+			approveExpected: false,
+			expectedResult: ctrl.Result{
+				RequeueAfter: time.Minute,
+			},
+			expectedStatus:      models.HostStatusInstalling,
+			expectedStage:       models.HostStageJoined,
+			clusterInstall:      newAciBaremetalWithoutMAPI("test-cluster-aci", testNamespace),
```

Comment:
Why does this say "WithoutMAPI" but the test name says "with MAPI"?

Reply:
Oops, the naming is incorrect, modified it!